app/lib/vendor_api/changes/add_interviews_to_application.rb

```diff
@@ -0,0 +1,11 @@
+module VendorAPI
+  module Changes
+    class AddInterviewsToApplication < VersionChange
+      description 'Include interviews in application json responses.'
+
+      resource InterviewPresenter
+      resource AddInterviewsToApplicationPresenter
```

Comment:
Presenter modules only nee to be configured as part of the presenter configuratio (similar to the next line), so this is not needed.